app/assets/javascripts/discourse/app/lib/api.js

```diff
@@ -0,0 +1,18 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+
+/**
+ * apiInitializer(version, apiCodeCallback, opts)
+ *
+ * An API to simplify the creation of initializers for plugins/themes by removing
+ * some of the boilerplate.
+ */
+let _apiInitializerId = 0;
+export function apiInitializer(version, cb, opts) {
+  return {
+    name: `api-initializer${_apiInitializerId++}`,
```

Comment:
Since there is no meaningful name used in an apiInitializer, will debugging errors in the console be more painful for customers when themes break?

Reply:
I'm pretty sure that name is never displayed anywhere. The stack trace will be to the actual module where the code lives.